code/modules/language/language_menu.dm

```diff
@@ -21,6 +21,7 @@
 /datum/language_menu/ui_data(mob/user)
 	var/list/data = list()
 
+	var/is_admin = check_rights_for(user.client, R_ADMIN)
```

Comment:
this should probably include R_DEBUG for seeing it, not granting though since they can't use the language menu for that